sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/text/TextSuite.scala

```diff
@@ -65,6 +65,14 @@ class TextSuite extends QueryTest with SharedSQLContext {
     }
   }
 
+  test("support for partitioned reading") {
+    val df = spark.read.format("text").load(testDir)
```

Comment:
this is probably ok because we still have some test cases that do this (otherwise we'd need to generate the data and then make sure the directory structure is valid first)